content/Classes/Classes.q.md

```diff
@@ -0,0 +1,1088 @@
+# Classes
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+    class MyClass 
+    {
+        public $myProp = "This is a class property.";
+        
+        public function myMethod() 
+        {
+            echo "This is a class method.";
+        }
+    }
+?>
+```
+/// type=SS, answer=[3]
+
+Execute the program. What is its output?
+
+ - It prints `MyClass`.
+
+ - It produces an error.
+
+ - No output is displayed.
+
+ - It prints `This is a class method.`.
+
+ - It prints `This is a class property.`.
+
+
+/// type=SS, answer=[4]
+
+Why is there no output displayed?
+
+ - `MyClass` is not a valid class name.
+
+ - There are no parentheses `()` after `MyClass`.
+
+ - `$myProp` is not a valid property of `MyClass`.
+
+ - There is no `MyClass` instance created at the moment.
+
+ - On line 6, the `myMethod()` method definition is invalid.
+
+
+/// type=SS, answer=[3]
+
+On line 2, what is `class`?
+
+ - It is a string.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a variable.
+
+ - It is a property.
+
+
+/// type=SS, answer=[5]
+
+On line 2, what is `MyClass`?
+
+ - It is a string.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a class name.
+
+
+/// type=SS, answer=[4]
+
+On line 3, what does the open curly brace `{` indicate?
+
+ - The end of the program.
+
+ - The beginning of the program.
+
+ - The end of the `class` declaration.
+
+ - The beginning of the `class` declaration.
+
+
+/// type=SS, answer=[3]
+
+On line 10, what does the close curly brace `}` indicate?
+
+ - The end of the program.
+
+ - The beginning of the program.
+
+ - The end of the `class` declaration.
+
+ - The beginning of the `class` declaration.
+
+
+/// type=SS, answer=[3]
+
+On line 4, what is `public`?
+
+ - It is a string.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a class name.
+
+
+/// type=SS, answer=[4]
+
+On line 4, what is `$myProp`?
+
+ - It is a string.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a class name.
+
+
+/// type=SS, answer=[1]
+
+In the statement `$myProp = "This is a class property."` on line 4, what is `"This is a class property."`?
+
+ - It is a string.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a class name.
+
+
+/// type=MS, answer=[1,2]
+
+Which statements correctly describe the code on line 4?
+
+ - It is the definition of the `$myProp` property.
+
+ - It declares and initializes the `$myProp` property of `MyClass`.
+
+ - It accesses the string `This is a class property.` of the `$myProp` property.
+
+ - It displays the string `This is a class property.` of the `$myProp` property.
+
+ - It removes the string `This is a class property.` from the `$myProp` property.
+
```

Comment:
I think we have to retain the first statement since this describes the given code. It is not required to make the statements similar at all times. Please refer to this example. 
![image](https://user-images.githubusercontent.com/25945790/59832320-2a808300-9376-11e9-8c45-242f92d3cd9c.png)